editor/components/block-list/style.scss

```diff
@@ -931,3 +931,24 @@
 		content: '←';
 	}
 }
+
+.editor-block-list__block {
+	@include break-small {
+		// use a wider available space for hovering/selecting on top level blocks
```

Comment:
These should use capital "U" and have a period at the end of the sentence 🙂

Reply:
That said, I'm not sure what this (or the comment below) mean.

Reply:
Hi, @tofumatt thank you for your review. I updated the comments, let me know if you find the new version of the comments more helpful :)